editor/frontend/src/public/locales/en/Editor.json

```diff
@@ -65,6 +65,69 @@
     "Redo": {
       "keyin": "editor redo",
       "flyover": "Redo Elements"
+    },
+    "ProjectLocation": {
+      "Show": {
+        "keyin": "editor project location show",
+        "flyover": "Show Project Location"
+      },
+      "Hide": {
+        "keyin": "editor project location hide",
+        "flyover": "Hide Project Location"
+      },
+      "Cancel": {
+        "keyin": "editor project location cancel",
+        "flyover": "Abandon Project Location Changes"
+      },
+      "Save": {
+        "keyin": "editor project location save",
+        "flyover": "Save Project Location Changes"
+      },
+      "Geolocation": {
+        "Move": {
+          "keyin": "editor project geolocation move",
+          "flyover": "Move Project Geolocation"
+        },
+        "Point": {
+          "keyin": "editor project geolocation point",
+          "flyover": "Define Project Geolocation"
+        },
+        "North": {
+          "keyin": "editor project geolocation north",
+          "flyover": "Define Project North"
+        }
+      },
+      "Prompts": {
+        "IdentifyHandle": "Identify project extents or handle",
+        "IdentifyRefPoint": "Identify reference point",
+        "DefineAngle": "Enter point to define angle",
+        "DefineOffset": "Enter point to define offset",
+        "IdentifyKnownLocation": "Identify location of known coordinate",
+        "ConfirmCoordinates": "Confirm coordinates",
+        "SpecifyCoordinates": "Specify coordinates then accept"
+      },
+      "Inputs": {
+        "AcceptHandle": "Accept project extents or handle"
+      },
+      "Message": {
+        "NotActive": "Geolocation tool is not enabled for a spatial view",
+        "NotAllowed": "A valid projected coordinate system already exists",

```

Comment:
"projected coordinate system" -> "Geographic projection system"? Ask @AlainRobertAtBentley

Reply:
Will change to "geographic coordinate reference system" that's what was used in docs...

Reply:
There are many conventions for naming. Within MicroStation we usually use Geographic Coordinate System which can be of two types Projected and non-projected(Lat/Long). OrbitGT, ESPG and others use Coordinate Reference System. In iModelJS I introduced the more explicit name Geographic Coordinate Reference System which is a both more explicit and more conventional so your change is valid @bbastings